governance/votes/Executive vote - November 1, 2023.md

```diff
@@ -0,0 +1,92 @@
+
+If this executive proposal passes, a one-time MKR payment in the amount of **6.34 MKR** will be transferred to the Core Unit's Auditor Wallet ([0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4](https://etherscan.io/address/0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4)).
+
+### Add GUSD and USDP Jar and SwapInputConduit Contracts to Chainlog
+
+*Proposal*: [Forum Post](https://forum.makerdao.com/t/proposed-housekeeping-item-upcoming-executive-spell-2023-11-01/22477) "Proposed Housekeeping Item (Upcoming Executive Spell - 2023-11-01)"\
+*Authorizing Party*: [Protocol Scope Facilitator](http://forum.makerdao.com/t/proposed-housekeeping-item-upcoming-executive-spell-2023-11-01/22477/2)
+
+In [December 2022](https://forum.makerdao.com/t/gusd-institutional-rewards-technical-solution-risk-assessment-and-deployment-details-for-receiving-payments-of-rewards/19055), a GUSD & MakerDAO partnership for institutional rewards was rolled out. In order for MakerDAO to receive the GUSD rewards, two contracts were deployed: the RwaSwapInputConduit2 and RwaJar. Recently, MakerDAO entered into a similar [partnership with Trident](https://vote.makerdao.com/executive/template-executive-vote-usdp-psm-incentives-reth-initial-offboarding-rwa-vaults-reconfiguration-various-parameter-changes-avc-and-ad-compensation-facilitator-and-ecosystem-actor-compensation-spark-proxy-spell-october-11-2023) to receive USDP rewards. This deal also requires deploying the above-mentioned contracts.
+
+If this executive proposal passes, these contracts will be added to the [Chainlog](https://chainlog.makerdao.com/) as:
+
+- [`MCD_PSM_GUSD_A_JAR`](https://etherscan.io/address/0xf2E7a5B83525c3017383dEEd19Bb05Fe34a62C27)
+- [`MCD_PSM_GUSD_A_INPUT_CONDUIT_JAR`](https://etherscan.io/address/0x6934218d8B3E9ffCABEE8cd80F4c1C4167Afa638)
+- [`MCD_PSM_PAX_A_JAR`](https://etherscan.io/address/0x8bF8b5C58bb57Ee9C97D0FEA773eeE042B10a787)
+- [`MCD_PSM_PAX_A_INPUT_CONDUIT_JAR`](https://etherscan.io/address/0xDa276Ab5F1505965e0B6cD1B6da2A18CcBB29515)
+
+Further, `<conduit>.rely(MCD_ESM)` will be called on the SwapInputConduit contracts to allow the ESM module to `deny` the pause proxy. In the event of a successful governance attack, this prevents funds from being stolen from the SwapInputConduit contracts. 
```

Comment:
@MakerJanSky , fyi, as of time of writing, this is still a proposed addition to the Spell sheet (added after Friday EOD deadline). waiting on confirmation from Sidestream team.